README.md

```diff
@@ -44,6 +48,9 @@
        1                                       10
 ```
 
+   Note: to use on Windows, install `Cygwin` and the `mingw64-x86_64-gcc-core` package,
```

Comment:
I could make it work easily with MSYS2, would it be useful to describe it?

Reply:
Please see [README.windows.md](https://github.com/JuliaLang/julia/blob/master/README.windows.md):

> The recommended way of compiling Julia from source on Windows is by cross compiling from Cygwin, using versions of the MinGW-w64 compilers available through Cygwin's package manager.

> Compiling Julia from source using MSYS2 has worked in the past but is not actively supported.

What would be the advantage of using `juliac` with `MSYS2`?

Reply:
none, it would just be an extra flexibility, but given that text on the windows README I agree that installing Cygwin should be the only recommendation.